composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/UserView.kt

```diff
@@ -57,66 +60,103 @@ fun UserView(userViewModel: UserViewModel, onLogOut: () -> Unit) {
                 contentDescription = null
             )
             Spacer(modifier = Modifier.height(36.dp))
```

Comment:
Good that this is scrollable now, might make sense to just reduce this spacer to 16.dp since only one button is cut off currently.

Reply:
Profile spacing updated.